.gitignore

```diff
@@ -15,6 +15,7 @@ Build/Visual Studio 2017/Debug
 
 # cmake
 out
```

Comment:
this is what i think is a default output folder for Visual Studio (its /out/build  in fact), could you please add a comment here that that's for VS?